src/hal/co_if_can.h

```diff
@@ -175,6 +175,44 @@ extern "C" {
         (f)->Data[((p)+3)&0x7] = (uint8_t)(((uint32_t)(n)) >> 24); \
     } while(0)
 
+/*! \brief GET DATA LONG LONG
+*
+*    This macro extracts a data long long (64 bit) out of the CAN frame.
+*
+* \param f
+*    The CAN frame
+*/
+#define CO_GET_LONGLONG(f)  \
```

Comment:
a static inline function is much safer than a macro and optimizes away just as cleanly without function call overhead.